examples/shaders/sky-shader/src/lib.rs

```diff
@@ -175,3 +175,14 @@ pub fn main_vs(#[spirv(vertex_index)] vert_idx: i32, #[spirv(position)] builtin_
 
     *builtin_pos = pos.extend(0.0).extend(1.0);
 }
+
+#[cfg(test)]
+mod tests {
+    // Note this useful idiom: importing names from outer (for mod tests) scope.
+    use super::*;
+
+    #[test]
+    fn test_tonemap() {
+        assert_eq!(tonemap(vec3(1_f32,1_f32,1_f32)), vec3(0.001261625, 0.001261625, 0.001261625));
+    }
+}
```

Comment:
nit: we generally include newlines at the end of files in this project, so consistency is nice to have. I believe this will be automatically fixed once you run `cargo fmt` (as the test needs formatting)!